x-pack/plugins/cases/public/components/user_actions/markdown_form.tsx

```diff
@@ -43,70 +28,21 @@ const UserActionMarkdownComponent = forwardRef<
   UserActionMarkdownProps
 >(({ id, content, caseId, isEditable, onChangeEditable, onSaveContent }, ref) => {
   const editorRef = useRef();
-  const initialState = { content };
-  const { form } = useForm<Content>({
-    defaultValue: initialState,
-    options: { stripEmptyFields: false },
-    schema,
-  });
-
   const fieldName = 'content';
-  const { appId } = useCasesContext();
-  const draftStorageKey = getMarkdownEditorStorageKey(appId, caseId, id);
-  const { setFieldValue, submit } = form;
-
-  const handleCancelAction = useCallback(() => {
-    onChangeEditable(id);
-    removeItemFromSessionStorage(draftStorageKey);
-  }, [id, onChangeEditable, draftStorageKey]);
-
-  const handleSaveAction = useCallback(async () => {
-    const { isValid, data } = await submit();
-
-    if (isValid && data.content !== content) {
-      onSaveContent(data.content);
-    }
-    onChangeEditable(id);
-    removeItemFromSessionStorage(draftStorageKey);
-  }, [content, id, onChangeEditable, onSaveContent, submit, draftStorageKey]);
-
-  const setComment = useCallback(
-    (newComment) => {
-      setFieldValue(fieldName, newComment);
-    },
-    [setFieldValue]
-  );
-
-  useImperativeHandle(ref, () => ({
-    setComment,
-    editor: editorRef.current,
-  }));
 
   return isEditable ? (
-    <Form form={form} data-test-subj="user-action-markdown-form">
-      <UseField
-        path={fieldName}
-        component={MarkdownEditorForm}
-        componentProps={{
-          ref: editorRef,
-          'aria-label': 'Cases markdown editor',
-          value: content,
-          id,
-          draftStorageKey,
-          bottomRightContent: (
-            <UserActionMarkdownFooter
-              handleSaveAction={handleSaveAction}
-              handleCancelAction={handleCancelAction}
-            />
-          ),
-          initialValue: content,
-        }}
-      />
-    </Form>
+    <EditableMarkdown
```

Comment:
I think we should pass the `ref` (`ref={ref}`).

Reply:
Then we will need a type conversion. Because `EditableMarkdown` expects `React.MutableRefObject` not `React.ForwardedRef`

Reply:
I tested and I did not get an error. What I mean is

```
<EditableMarkdown
      id={id}
      content={content}
      caseId={caseId}
      onChangeEditable={onChangeEditable}
      onSaveContent={onSaveContent}
      editorRef={editorRef} <-- I did not tough this
      fieldName={fieldName}
      formSchema={schema}
      ref={ref} <-- here is the change
    />
```